src/near.ts

```diff
@@ -85,7 +85,7 @@ export class Near {
         this.connection = Connection.fromConfig({
             networkId: config.networkId,
             provider: { type: 'JsonRpcProvider', args: { url: config.nodeUrl, headers: config.headers } },
-            signer: config.signer || { type: 'InMemorySigner', keyStore: config.keyStore || config.deps.keyStore }
+            signer: config.signer || { type: 'InMemorySigner', keyStore: config.keyStore || (config.deps && config.deps.keyStore) }
```

Comment:
Hmm, I have just taken a look and it seems like `NearConfig.deps` is deprecated and we should use `NearConfig.keyStore` to access key store anyway. Am I missing something here?

Reply:
Yes, it's deprecated, but the keyStore is marked optional in our docs. And it was optional all the time. But now it works only if you are providing `{}`. If you will provide `undefined`, we will have a crash on `config.deps.keyStore` because `config.deps` is undefined.

Reply:
Oh, I see. I was confused about where the `config.keyStore` was coming from. Since it is supplied by the user, your change makes sense to me.